tap_zendesk/streams.py

```diff
@@ -554,6 +555,38 @@ def sync(self, state): # pylint: disable=unused-argument
         for policy in self.client.sla_policies():
             yield (self.stream, policy)
 
+
+class Calls(Stream):
+    name = "calls"
+    replication_method = "INCREMENTAL"
+
+    def sync(self, state):
+        # The incremental Talk endpoint isn't currently supported by the Zenpy
+        # library, though there is an open PR for getting that in there:
+        # https://github.com/facetoe/zenpy/pull/454
+        # If/when that gets merged we can update, but for now we have this!
+
+        bookmark = self.get_bookmark(state)
+        bookmark = round(bookmark.timestamp())
```

Comment:
Very minor nitpick: using `math.floor` might be better to ensure there are no subsecond gaps between requests (that's assuming we are requesting more recent items with each page).